DeepSpeech.py

```diff
@@ -329,7 +336,7 @@ def log_variable(variable, gradient=None):
     It logs scalar values for the mean, standard deviation, minimum and maximum.
     Furthermore it logs a histogram of its state and (if given) of an optimization gradient.
     '''
-    name = variable.name
+    name = variable.name.replace(':', '_')
```

Comment:
Why this change?

Reply:
':' is an illegal character in TensorBoard summary names so it produces a ton of warnings.

Reply:
(I described the reasoning in the commit message)